### ISSUE TITLE
Fix breadcrumbs for resource pages globally

### ISSUE THREAD

**Issue by MeghnaTewari** [hlxsites/moleculardevices#1041]
https://www.moleculardevices.com/en/assets/tutorials-videos/dd/img/automation-high-content-imaging-for-organoid-research

Should be  Home > Resources

![image](https://github.com/hlxsites/moleculardevices/assets/138755144/fcdc6b27-8952-48b2-ab70-66ebfcadcd8d)

**Comment by alexcarol** [comment]
How can we identify pages as "resource page"?

**Comment by MeghnaTewari** [comment]
@alexcarol Let's discuss this with Rajneesh in today's dev call.

**Comment by andreituicu** [comment]
@alexcarol these are the types which map to `Resources`: https://github.com/hlxsites/moleculardevices/blob/main/blocks/resources/resource-mapping.js . I think @duynguyen could share a bit more if you have additional questions.

**Comment by lydiapuric** [comment]
@MeghnaTewari @dev-rajneeshkumar

Could you please clarify for which types of Resources specifically we need to update the breadcrumb behaviour? It looks like just doing it for all resources does not match the drupal implementation.

It looks like the Lab Notes although they are resources, on the drupal implementation they are using the usual style of breadcrumbs.
https://live-md30.pantheonsite.io/lab-notes/microplate-readers/gxp-regulated-industry-assessments-of-microplate-readers
https://1041-breadcrumb--moleculardevices--hlxsites.hlx.page/lab-notes/microplate-readers/gxp-regulated-industry-assessments-of-microplate-readers

Also it is the same situation as with lab notes, although news are Resources according to moldev, they had the normal breadcrumb on the drupal site: https://live-md30.pantheonsite.io/newsroom/news/molecular-devices-launches-custom-organoid-line-expansion-service-using-unique-bioprocess-technology-for-high-throughput-applications

Currently following types are defined as Resources, see https://github.com/hlxsites/moleculardevices/blob/main/blocks/resources/resource-mapping.js. We can exclude complete types of resources types to get the Resources breadcrumb. Please define which of the Resource types should be excluded.

**Comment by lydiapuric** [comment]
@MeghnaTewari Sorry, I don't understand your answer. We are looking for a way to determine and categorize the type of pages where the page gets Home > Resources breadcrumb. 

Currently, the solution is based on the following Resource Type categories. All pages with these types get the Home > Resource breadcrumb! Please restrict the list as next step if this is the way to go.

- [x] Application NoteBlog
- [x] Brochure
- [x] Cell Counter
- [x] Citation
- [x] Customer Breakthrough
- [x] Data Sheet
- [x] eBook
- [x] Flyer
- [x] Infographic
- [x] News
- [x] Presentations
- [x] Product Insert
- [x] Publication
- [x] Scientific Poster
- [x] SDS
- [x] Technical Guide
- [x] Training Material
- [x] User Guide
- [x] Videos and Webinars
- [x] White Paper

**Comment by MeghnaTewari** [comment]
@lydiapuric Looks good!

Can we remove brochure text from the header?
Brochure: https://1041-breadcrumb--moleculardevices--hlxsites.hlx.page/en/assets/brochures/dd/img/a-life-sciences-company
![image](https://github.com/hlxsites/moleculardevices/assets/138755144/8839f7ed-61a4-43b4-91d9-326d807aa4b2)

**Comment by lydiapuric** [comment]
@MeghnaTewari > Can we remove brochure text from the header? - DONE!
